### DISCUSSION TITLE
REXENSE RH0039

### DISCUSSION THREAD

**Discussion by JoostV** [Koenkk/zigbee2mqtt#23322] [Devices]
I have a new zigbee device, but it is already registered as an other device.
It works out of the box, but it needs to be properly added.

This is the generated_external_definition
```
const {identify} = require('zigbee-herdsman-converters/lib/modernExtend');
const {philipsLight} = require('zigbee-herdsman-converters/lib/philips');

const definition = {
    zigbeeModel: ['RH0039'],
    model: 'RH0039',
    vendor: 'REXENSE',
    description: 'Automatically generated definition',
    extend: [identify(), philipsLight()],
    meta: {},
};

module.exports = definition;
```

This is the same product with a different name:
https://www.zigbee2mqtt.io/devices/322054.html

Can this be added properly, so it is not added double?

**Comment by Koenkk** [comment]
Added, thanks!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Comment by JoostV** [comment]
It works!
![image](https://github.com/user-attachments/assets/6ccbeb8a-6a9c-4b3a-a5c8-f7aab96caa52)

It can be added to the main branch.
